src/index.js

```diff
@@ -70,6 +70,60 @@ function saveToFile(filename, data) {
   fs.writeFileSync(filename, data);
 }
 
+/**
+ * Generate qrcodes from teh accounts that can be scanned with an authenticator app
+ * @param accounts A list of the auth accounts
+ */
+function saveToQRCodes(accounts){
+  
+  const QRCode = require('qrcode')
+  const fs = require("fs");
+  
+  const directory = "./qrCodes"
+  if(!fs.existsSync(directory)){
+    fs.mkdirSync(directory)
+  }
+
+  accounts.forEach(account => {
+    const name = account.name
+    const issuer = account.issuer
+    const secret = account.totpSecret
+    
+    const url = `otpauth://totp/${encodeURI(name)}?secret=${encodeURI(secret)}&issuer=${encodeURI(issuer)}`
+    const file = `${directory}/${issuer}(${name}).png`
+  
+    if(fs.existsSync(file)) {
+      console.log(`${file.yellow} already exists.`)
+    }else{
+      QRCode.toFile(file, url, (error) => {
+        if(error != null){
+          console.log(`Something went wrong while creating ${file}`, error)
+        }
+        console.log(`${file.green} created.`)
+      })
+    }
+    
+  })
+}
+
+/**
+ * Saves to jason if the user said yes.
+ * @param promptResult The results from the promtg given to the user.
+ * @param accounts A list of the auth accounts.
+ */
+function toJson(promptResult, accounts) {
```

Comment:
because js is shit vs Ts, it may be cleaner to send each param individually instead of as a option object.
So `filename` instead of `prompResult`. Also, do the `promptResult.saveToFile.toLowerCase().startsWith("y")` check in `promptUserForUri` instead, and call a `saveToQrFile` method instead of using `toJson`. Create a `printAccounts` when just logging. `toJson` makes little sense as the method never returns any json.